src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
I see, thanks. So why is it easier for the sender to fill in the sourceAccount on the first payment than the destinationAccount on the last payment?